packages/components/Checkbox/src/CheckboxTokens.android.ts

```diff
@@ -3,27 +3,32 @@ import { TokenSettings } from '@fluentui-react-native/use-styling';
 import { globalTokens } from '@fluentui-react-native/theme-tokens';
 import { CheckboxTokens } from './Checkbox.types';
 
-// No Tokens for Sizes , hovered, pressed and circular checkbox as they're not supported for Android Platform
+// No Tokens for Sizes , hovered and circular checkbox as they're not supported for Android Platform
 
 export const defaultCheckboxTokens: TokenSettings<CheckboxTokens, Theme> = (t: Theme) =>
   ({
     checkboxBorderWidth: globalTokens.stroke.width15,
     checkboxBorderRadius: globalTokens.corner.radius40,
-    checkboxSize: globalTokens.font.size600,
-    checkmarkSize: globalTokens.font.size100,
+    checkboxSize: globalTokens.size200,
+    checkmarkSize: globalTokens.size120,
+    rippleColor: '#D4D4D4',
```

Comment:
Does this color not have a token?

Reply:
@rurikoaraki , For the ripple color value on Android, we have already notified design. 
They have work item for the ripple color that best fit on dark mode and light mode and releasing a separate token for it.  
For now , We will be using this hardcoded value. There are few places such as Button ripple as well , where this needs to be added.

Reply:
Should we file a Github issue and put it in comments?

Reply:
I have raised a issue for it here , will pickup once tokens are in : 
https://github.com/microsoft/fluentui-react-native/issues/2429